gazetteer-style.cpp

```diff
@@ -163,6 +163,25 @@ gazetteer_style_t::flag_t gazetteer_style_t::parse_flags(std::string const &str)
     return out;
 }
 
+bool gazetteer_style_t::add_metadata_style_entry(std::string const &key,
+                                                 std::string const &value)
```

Comment:
'value' is unused here. You can just remove the second parameter. (I wonder why travis does not complain, we need to change that.)